takecare19/takecare19
# Display 20 resources

It took me a while but i finally figured it out 😅 Once we add loading states I think it will be really nice!

### Testing instructions
* Load the page, make sure only 20 resources are displayed
* Make sure the "Load more" button works
* Switch categories, make sure the right resources are displayed and "Load more" only loads more of that category
* Make sure the "Load more" button is disabled and says "End of list" if there are no more resources
